cmd/ratify/cmd/serve.go

```diff
@@ -62,6 +64,8 @@ func NewCmdServe(argv ...string) *cobra.Command {
 	flags.BoolVar(&opts.enableCrdManager, "enable-crd-manager", false, "Start crd manager if enabled")
 	flags.IntVar(&opts.cacheSize, "cache-size", httpserver.DefaultCacheMaxSize, "Cache size for the verifier http server")
 	flags.DurationVar(&opts.cacheTTL, "cache-ttl", httpserver.DefaultCacheTTL, "Cache TTL for the verifier http server")
+	flags.StringVar(&opts.metricsType, "metrics-type", httpserver.DefaultMetricsType, "Metrics exporter type to use")
```

Comment:
should we mention what is the default value in the help message? do we have any docs for parameters of the cmd command?